docs/getting-started/integration.md

```diff
@@ -15,87 +15,82 @@ Chart.js can be integrated with plain JavaScript or with different module loader
 
 Chart.js 3 is tree-shakeable, so it is necessary to import and register the controllers, elements, scales and plugins you are going to use.
 
-For all available imports see the example below.
+### Quick start
+
+If you don't care about the bundle size, you can use the `auto` package ensuring all features are available:
 
 ```javascript
-import {
-  Chart,
-  ArcElement,
-  LineElement,
-  BarElement,
-  PointElement,
-  BarController,
-  BubbleController,
-  DoughnutController,
-  LineController,
-  PieController,
-  PolarAreaController,
-  RadarController,
-  ScatterController,
-  CategoryScale,
-  LinearScale,
-  LogarithmicScale,
-  RadialLinearScale,
-  TimeScale,
-  TimeSeriesScale,
-  Decimation,
-  Filler,
-  Legend,
-  Title,
-  Tooltip,
-  SubTitle
-} from 'chart.js';
-
-Chart.register(
-  ArcElement,
-  LineElement,
-  BarElement,
-  PointElement,
-  BarController,
-  BubbleController,
-  DoughnutController,
-  LineController,
-  PieController,
-  PolarAreaController,
-  RadarController,
-  ScatterController,
-  CategoryScale,
-  LinearScale,
-  LogarithmicScale,
-  RadialLinearScale,
-  TimeScale,
-  TimeSeriesScale,
-  Decimation,
-  Filler,
-  Legend,
-  Title,
-  Tooltip,
-  SubTitle
-);
-
-const myChart = new Chart(ctx, {...});
+import Chart from 'chart.js/auto';
 ```
 
-A short registration format is also available to quickly register everything.
+Or if you are planning to optimize the bundle later, you can start by importing and registering `registerables`, which also ensures all features are available:
 
 ```javascript
 import { Chart, registerables } from 'chart.js';
 Chart.register(...registerables);
 ```
 
```

Comment:
I feel like it just makes it more complicated by documenting more ways to do the same thing. We should either document this or auto, but not both. I'd be fine dropping either
```suggestion
```